WordPress/gutenberg
# added width style for align center issue in image block

## What?
I have added the width:100% in .wp-block-image.aligncenter class hierarchy. So by doing this now image is align in center by older CSS text-align:center; 

## Why?
So the problem was when we add image in the group block then we are not able to align the image in the center. So the issue was mentioned in this ticket(link: https://github.com/WordPress/gutenberg/issues/39291) and I solved this issue by this width css.

## How?

The implementation details are very simple just open the file style.scss(path: packages/block-library/src/image/style.scss) and add the width 100% in .wp-block-image.aligncenter class hierarchy.

## Testing Instructions
1. Open a Post or Page in theme Twenty Twenty-Two.
2. Add a group block, set some background color
3. Add an image block into the group block
4. Resize the image so it's smaller than the group block width
5. Change image alignment to center
6. Notice that now the image is centered in front and editor side both.

## Screenshots or screencast 
Image of editor side link: https://prnt.sc/vD8DpJWDgZNy
Image of front side link: https://prnt.sc/LZhJoIWS3m1S
